### ISSUE TITLE
Config on web site

### ISSUE THREAD

**Issue by tomaszcz123** [Edzelf/ESP32Radio-V2#136]
I have a little problem. I cannot configure it via the website. I only have this.
![Screenshot_1](https://github.com/Edzelf/ESP32Radio-V2/assets/156236250/2379260a-0278-463c-8519-7ed295bbfc86)

**Comment by Edzelf** [comment]
Give some more info.  What happens if you press the "Default" button?

**Comment by tomaszcz123** [comment]
You're right. Everything works correctly. I spend too long in front of the computer :) Thank you.
